app/views/ViewProvider.scala

```diff
@@ -23,100 +23,41 @@ import play.api.mvc.Request
 import play.twirl.api.HtmlFormat
 import viewModels.Section
 import views.html._
-import views.html.v2._
 
 import java.util.UUID
 import javax.inject.Inject
 
-sealed trait ViewProvider {
-
-  def guaranteeReferenceNumberView(
-    form: Form[String],
-    mode: Mode
-  )(implicit request: Request[?], messages: Messages): HtmlFormat.Appendable
-
-  def accessCodeView(
-    form: Form[String],
-    mode: Mode
-  )(implicit request: Request[?], messages: Messages): HtmlFormat.Appendable
-
-  def checkYourAnswersView(sections: Seq[Section])(implicit request: Request[?], messages: Messages): HtmlFormat.Appendable
-
-  def balanceConfirmationView(
-    balance: String,
-    timestamp: Timestamp,
-    referral: Option[String]
-  )(implicit request: Request[?], messages: Messages): HtmlFormat.Appendable
-
-  def couldNotCheckBalance(balanceId: Option[UUID])(implicit request: Request[?], messages: Messages): HtmlFormat.Appendable
-}
-
-class V1ViewProvider @Inject() (
+class ViewProvider @Inject() (
```

Comment:
This view provider class is now also pretty obsolete. Can inject the relevant view directly into the relevant controller.